packages/apps-config/src/ui/logos/index.ts

```diff
@@ -124,6 +125,7 @@ export const chainLogos: Record<string, unknown> = [
   ['Beast Developer', nodeBeast],
   ['Crust PC1', nodeCrust],
   ['ChainX', nodeChainx],
+  ['Charcoal Testnet', chainAltair],
```

Comment:
Do we need to add altair here linked to chainAltair? In addition to adding charcoal to `nodeCentrifuge`?

Reply:
Not sure if it's needed, but it can't hurt.